### PR TITLE
#114 - Add NullAnonymizer

### PR DESCRIPTION
**Pull request by SimonMellerin**

### CODE REVIEW

File: src/Anonymization/Anonymizer/Core/NullAnonymizer.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    /**
+     * @inheritdoc
+     */
+    public function anonymize(Update $update): void
+    {
+        $expr = $update->expression();
+        $update->set(
+            $this->columnName,
+            $expr->raw('NULL'),
```
Comment (pounard) [inline src/Anonymization/Anonymizer/Core/NullAnonymizer.php:26]: Just write `null` here, no need to use an expression instance.

File: docs/content/anonymization/core-anonymizers.md [hunk 2]
```diff
@@ -265,6 +265,41 @@ customer:
 ```
 :::
 
+## NullAnonymizer
+
+Set all value to `NULL`.
```
Comment (Lonnytunes) [inline docs/content/anonymization/core-anonymizers.md:270]:
```suggestion
Set all values to `NULL`.
```

File: src/Anonymization/Anonymizer/Core/NullAnonymizer.php (hunk 1 above)
Comment (SimonMellerin) [reply src/Anonymization/Anonymizer/Core/NullAnonymizer.php:26]: it does not work with just 'null'

Comment (SimonMellerin) [reply src/Anonymization/Anonymizer/Core/NullAnonymizer.php]: I updated with `$expr->null()`

File: src/Anonymization/Anonymizer/Core/NullAnonymizer.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+{
+    /**
+     * @inheritdoc
+     */
+    public function anonymize(Update $update): void
+    {
+        $expr = $update->expression();
+        $update->set(
+            $this->columnName,
+            $expr->null(),
```
Comment (pounard) [inline src/Anonymization/Anonymizer/Core/NullAnonymizer.php:26]: You can simply pass `null` here, instead of using an expression.

Comment (SimonMellerin) [reply src/Anonymization/Anonymizer/Core/NullAnonymizer.php:26]: no, it does not work with just `null`, I tested

**Review by Lonnytunes — APPROVED** [review]

**Pull request merged**